chats/apps/projects/consumers/template_type_consumer.py

```diff
@@ -17,6 +20,11 @@ def consume(message: amqp.Message):
         print(f"[TemplateTypeConsumer] - Consuming a message. Body: {message.body}")
         body = JSONParser.parse(message.body)
 
+        try:
+            Project.objects.get(uuid=body.get("project_uuid"))
+        except Exception as err:
+            raise InvalidProjectData(err)
```

Comment:
In this context it makes more sense to raise the exception `InvalidTemplateTypeData` instead `InvalidProjectData`!